.github/workflows/build-publish.yml

```diff
@@ -113,7 +113,36 @@ jobs:
 
   natives-linux:
     runs-on: ubuntu-20.04
+    container:
+      image: ubuntu:18.04
     steps:
+      - name: Install dependencies into minimal dockerfile
+        run: |
+          # ubuntu dockerfile is very minimal (only 122 packages are installed)
+          # need to install updated git (from official git ppa)
+          apt update
+          apt install -y software-properties-common
+          add-apt-repository ppa:git-core/ppa -y
+          # install dependencies expected by other steps
+          apt update
+          apt install -y git \
+          curl \
+          ca-certificates \
+          wget \
+          bzip2 \
+          zip \
+          unzip \
+          xz-utils \
+          openjdk-11-jdk-headless \
```

Comment:
```suggestion
```

Reply:
We could also just use a Docker image with java pre-installed https://hub.docker.com/_/eclipse-temurin

Reply:
Yes, but then it gets inconsistent with the rest of the gha, which in my opinion is better to avoid.

Reply:
comitted the suggestion. you can just squash while merging